docs/contributors_guide/maintenance/Tagging_and_Releasing_Components.rst

```diff
@@ -0,0 +1,43 @@
+Tagging and Releasing Components
+================================
+
+This section describes the release procedures for SIMP.  The SIMP
+Team releases:
+
+* Individual Puppet modules as tar files to `PuppetForge`_
+* Individual Puppet modules as signed RPMs to `packagecloud`_
```

Comment:
@op-ct Didn't the bullets that followed  cover the other components?  Don't understand the comment.